docs/UserGuide.md

```diff
@@ -342,6 +358,7 @@ _Details coming soon ..._
 | **Edit**       | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                            |
 | **Find**       | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                            |
 | **Delete**     | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                   |
+| **Deletetag**  | `deletetag INDEX t/[TAG]` <br> e.g. `deletetag 1 t/Client`                                                                                                            |
```

Comment:
Actually is there a reason why the name of the action can't be `Delete Tag` with a space?

Reply:
![image](https://github.com/user-attachments/assets/9767a637-9f75-4be4-9ed8-2d468c5f9e7d)

From what I understand currently the `parseCommand` method us using a class called matcher and is using a function called group which take the first word as the command phrase then tag will be deemed as another input.

Reply:
As in, the Deletetag in the "Action" part of the table could just be the name of the command right? The name of the action is to delete (a) tag. But the command word is `deletetag`